EnglishFilter/sections/css_extended.txt

```diff
@@ -10,6 +10,8 @@ duplichecker.com#?#.side-bar > div.search-panel-sec:has(> div > script[src^="htt
 ! https://github.com/AdguardTeam/AdguardFilters/issues/44345
 gadgetsnow.com#?#.clearfix > div[class*=" "]:has(> div > div > a[onclick])
 gadgetsnow.com#?#.sidebar > .clearfix > div[class*=" "]:has(> div > div > div > a[onclick])
+! https://github.com/AdguardTeam/AdguardFilters/issues/44836
+official-amtemu.com##aside.sidebar > div.theiaStickySidebar > div#text-html-widget-2:has(div > ins.adsbygoogle)
```

Comment:
It's better re-write rule to this `official-amtemu.com##.theiaStickySidebar > div.widget:has(> div > ins.adsbygoogle)`